cycode/cli/files_collector/sca/maven/restore_maven_dependencies.py

```diff
@@ -15,11 +15,12 @@
 BUILD_MAVEN_FILE_NAME = 'pom.xml'
 MAVEN_CYCLONE_DEP_TREE_FILE_NAME = 'bom.json'
 MAVEN_DEP_TREE_FILE_NAME = 'bcde.mvndeps'
+OUTPUT_FILE_MANUALLY = False
```

Comment:
remove as discussed in prev PR; we made false by default

Reply:
done